liquibase-standard/src/main/java/liquibase/statement/core/TableIsEmptyStatement.java

```diff
@@ -8,7 +8,17 @@
 @AllArgsConstructor
 public class TableIsEmptyStatement extends AbstractSqlStatement {
 
-    private final String catalogName;
-    private final String schemaName;
-    private final String tableName;
+    private DatabaseTableIdentifier databaseTableIdentifier = new DatabaseTableIdentifier(null, null, null);
```

Comment:
I think in this case having `DatabaseTableIdentifier` is not applicable here, since we are using `Lombok` constructor annotation, and getter methods are not needed in this case.